notebooks/From FHIR to ML - Getting Started with the PHC Python SDK.ipynb

```diff
@@ -0,0 +1,2264 @@
+{
```

Comment:
<p>Are we still advocating using <code>get_codes</code>  instead of the summary service? I don't have an opinion, just asking.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lifeomic/phc-examples/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>